ktor-server/ktor-server-core/common/test/io/ktor/server/application/ApplicationPluginTest.kt

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application
+
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.server.testing.*
+import io.ktor.util.*
+import io.ktor.utils.io.*
+import io.ktor.utils.io.core.*
+import kotlinx.coroutines.test.*
+import kotlin.test.*
+
+class ApplicationPluginTest {
+    @Test
+    fun test_empty_plugin_does_not_break_pipeline() = testApplication {
```

Comment:
Could you tell me the reason for `_` instead of camel case?

Reply:
ATM it was easier to migrate from ` `(space) to `-`(dash) automatically - if you have any idea on how to automate the migration to camel case, I would be glad to hear